New feature: model adaptivity (#198)
precice/micro-manager: docs/configuration.md

}
```

## Model Adaptivity

See the [model adaptivity](tooling-micro-manager-model-adaptivity.html) documentation for a detailed explanation about the interface.

To turn on model adaptivity, set `"model_adaptivity": true` in `simulation_params`. Then under `model_adaptivity_settings` set the following variables:

Parameter | Description
--- | ---

> `micro_file_names` | List of paths to the files containing the Python importable micro simulation classes. If the files are not in the working directory, give the relative path from the directory where the Micro Manager is executed. At least 2 files.

Reviewer: ```suggestion
`micro_file_names` | List of paths to the files containing the Python importable micro simulation classes. If the files are not in the working directory, give the relative path from the directory where the Micro Manager is executed. Requires a minimum of 2 files.
```

`switching_function` | Path to the file containing the Python importable switching function. If the file is not in the working directory, give the relative path from the directory where the Micro Manager is executed.

> Example of adaptivity configuration is

Reviewer: ```suggestion
Example of model adaptivity configuration is
```
